public/data/ruthiel.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Ruthiel",
+    "type": "personal",
+    "bio": "I like to code (professionaly and personaly), listen to jazz, playing guitar and other instruments, crossfit, podcasts!",
+    "avatar": "https://github.com/ruthiel.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/ruthiel",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/tofucru",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Reddit",
+        "url": "https://reddit.com/tofucru",
```

Comment:
**There is a broken link. Please update it with the correct one!** 
![image](https://user-images.githubusercontent.com/88786642/180671899-d342f8a0-973f-437b-8599-23f5f2f16c7c.png)


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Reply:
Oh!
Thank you for your review!
Fixed already!